atom/browser/web_contents_preferences.cc

```diff
@@ -309,6 +309,11 @@ void WebContentsPreferences::AppendCommandLineSwitches(
     command_line->AppendSwitchASCII(switches::kBackgroundColor, "#fff");
   }
 
+  // --offscreen
```

Comment:
This should apply to `<webview>`s inside OSR windows as well. Those are not marked as `offscreen` in the `webPreferences` automatically if the parent is `offscreen`.

Reply:
Can you suggest how I might write a test for this? I was playing around in Fiddle trying to trigger a context menu to appear in an offscreen window and I couldn't figure out how to get it working.